### PR TITLE
Add standalone grid example

### PR DESCRIPTION
**Pull request by jmacaluso711**
**Explain the details for making this change. What existing problem does the pull request solve?**
Adds a standalone grid example

**Related github/jira issue (required)**:
n/a

**Steps necessary to review your pull request (required)**:
- pull and run
- go to http://localhost:4300/ids-layout-grid/standalone-css
- check the example
- also added to the grid readme.

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

File: src/ids-layout-grid/README.md [hunk 1]
```diff
@@ -105,6 +114,7 @@ A layout grid is not on its own keyboard focusable and has no keyboard interacti
 ## Proposed Changes
 
 - May need to re-visit responsiveness as we begin to test complex layouts.
+- Add fixed unit sizind options for cells and rows.
```
Comment (tmcconechy) [inline src/ids-layout-grid/README.md:117]:
```suggestion
- Add fixed unit size and options for cells and rows.
```
?

Comment (jmacaluso711) [reply src/ids-layout-grid/README.md:117]: Looking to add a sizing option so you can set certain cells in pixels and some with relative sizing. So for example, a 3 column grid where the first cell is 300px, 2nd one is 1fr and the 3rd one is 200px

Comment (tmcconechy) [reply src/ids-layout-grid/README.md:117]: sure was just the word sizind wasnt sure if it was size AND options or sizing options

Comment (jmacaluso711) [reply src/ids-layout-grid/README.md:117]: Gotcha.. yeah, `sizing options`

**Review by tmcconechy — DISMISSED** [review]

**Comment by tmcconechy** [comment]
Any more to add on this one @jmacaluso711 ?

**Comment by jmacaluso711** [comment]
@tmcconechy good to go for now

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**